src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue

```diff
@@ -573,6 +573,8 @@ export default {
               this.wallet_truncate(this.schema.address)
             ]))
           }
+        } else {
```

Comment:
This bypasses the `validateAddress` check. You'll be better just to move the amount check I think

Reply:
It check first if there is a valid address. Then, if there was an `address` and `amount` it must be added to `recipients`, otherwise fill in the `recipientId` input.